editor/create_dialog.cpp

```diff
@@ -182,11 +182,13 @@ void CreateDialog::add_type(const String &p_type, HashMap<String, TreeItem *> &p
 	bool can_instance = (cpp_type && ClassDB::can_instance(p_type)) || ScriptServer::is_global_class(p_type);
 
 	TreeItem *item = search_options->create_item(parent);
+	item->set_text(0, p_type);
 	if (cpp_type) {
-		item->set_text(0, p_type);
+		item->set_tooltip(0, DTR(EditorHelp::get_doc_data()->class_list[p_type].brief_description));
 	} else {
+		item->set_tooltip(0, ScriptServer::get_global_class_path(p_type));
```

Comment:
This should be performed in the button callback. You are reducing the tooltip into a plain path.
You can retrieve `p_type` later by using `get_text(0)`.